test-tools/wamr-ide/README.md

```diff
@@ -68,7 +72,15 @@ docker load --input ./wasm-toolchain.tar
 docker load --input ./wasm-debug-server.tar
 ```
 
-##### 3.2 Build docker images on host
+##### 2.1.2 Download the VS Code extension installation file from the GitHub release
+
+From now on, for each GitHub release, we have the same version tagged zip/tar.gz file(e.g. decompress `wamr_ide-1.1.2.tar.gz` `wamr_ide-1.1.2.zip`), which contains .vsix VS Code extension installation file. You can find and easily download in the GitHub release.
```

Comment:
Do we need to change `wamr_ide-1.1.2.tar.gz` `wamr_ide-1.1.2.zip` to `wamr-ide-1.1.2.tar.gz` `wamr-ide-1.1.2.zip`?
Or change to `wamr-ide-1.2.0.tar.gz` `wamr-ide-1.2.0.zip` when we are to release 1.2.0?

Reply:
I will rephrase and be more clear. `wamr-ide-1.1.2.tar.gz` means to be an example. What I am just trying to say is that in the future(for example, there are multiple releases like 1.1.2, 1.1.3, and 1.2.0), users can choose to download which version they prefer